actions/call_api.py

```diff
@@ -1,10 +1,22 @@
 from lib.actions import ZabbixBaseAction
 from zabbix.api import ZabbixAPI
 
+import six
+
 
 class CallAPI(ZabbixBaseAction):
     def run(self, api_method, token, **params):
         # Initialize client object to connect Zabbix server
+
+        # Python doesn't let you modify the length of a dict or list while iterating
+        iterate_params = params.copy()
+
+        # Look for Empty strings and None values, but allow False
+        # If None or '', remove it from params.
+        for key, value in six.iteritems(iterate_params):
+            if key in params and not params[key] and not params[key] is False:
+                del params[key]
```

Comment:
This removes all values below, are these expected processing?

```Python
params = {
  'will_be_removed_1': 0,
  'will_be_removed_2': [],
  'will_be_removed_3': {},
}
```

Reply:
Good Catch, It should only remove a `None` ideally. The intention is to remove parameters that are defined in `params` but are set to None because no value was specified.